types/cronitor/index.d.ts

```diff
@@ -0,0 +1,113 @@
+
+    type MonitorType = "job" | "heartbeat" | "check";
+
+    interface CronitorConfig {
+        config?: string; // Path to config file
+        apiVersion?: string;
+        timeout?: number;
+        environment?: string;
+    }
+
+    /**
+     * https://cronitor.io/docs/telemetry-api#parameters
+     */
+    interface TelemetryEvent {
+        state: MonitorState; // Indicate a state change for a monitor
+        env?: string; // The environment the telemetry event is being sent from
+        message?: string; // A URL-encoded message of up to 2000 characters
+        metrics?: {
+            duration?: number;
+            count?: number;
+            error_count?: number;
+        };
+        host?: string; // The hostname of the server sending the telemetry event
+        series?: string; // A unique user-supplied ID to collate related pings
+        status_code?: number; // Exit code returned from a background job
+    }
+
+    /**
+     * https://cronitor.io/docs/monitor-api#attributes
+     */
+    interface MonitorAttributes {
+        type: MonitorType; // Monitor type (required)
+        key: string; // Monitor's unique identifier (required)
+        schedule?: string; // Schedule for monitoring
+        assertions?: string[]; // Array containing strings
+        group?: string; // Group name
+        metadata?: Record<string, any>; // JSON serializable key/value pairs
+        name?: string; // Display name of the monitor
+        note?: string; // Additional context/troubleshooting information
+        notify?: string[] | { alerts: string[]; events: Record<string, boolean> }; // Notification settings
+        paused?: boolean; // Monitoring/alerting status
+        platform?: string; // Platform information
+        grace_seconds?: number; // Number of seconds for grace period
+        tags?: string[]; // Array of tags
+        timezone?: string; // Timezone
+        schedule_tolerance?: number; // Schedule tolerance
+        failure_tolerance?: number; // Failure tolerance
+        created?: string; // Timestamp when the monitor was created
+        disabled?: boolean; // Current disabled status
+        initialized?: boolean; // Whether the monitor has received a telemetry event
+        passing?: boolean; // Whether the monitor is currently passing or failing
+        running?: boolean; // Whether a job is running (only applicable for type: job)
+        defaultName?: string; // Default name for provisioning
+        defaultNote?: string; // Default note for provisioning
+        realert_interval?: number | string; // Interval for follow-up alerts
+        request?: {
+            url: string;
+            method?: string;
+            body?: string;
+            headers?: Record<string, string>;
+            cookies?: Record<string, string>;
+            timeout_seconds?: number;
+            follow_redirects?: boolean;
+            verify_ssl?: boolean;
+        }; // required by type: check
+    }
+
+    class Monitor {
```

Comment:
`Monitor` and `Event` cannot be class exposed here as they are not exported by the library. You should move them outside the namespace.

```js
const x = require("cronitor");
console.log(x. Monitor )
console.log(x. Event )
```

```
> c:\temp\test>node a.js
undefined
undefined
```

Reply:
Right - this is fixed now